special-interest-groups/sig-k8s/roles-and-organization-management.md

```diff
@@ -0,0 +1,64 @@
+# SIG K8S Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the K8S SIG is to have the **Active Contributor** community role in the TiDB-Operator repository.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 4 PRs merged within 2 months.
```

Comment:
In here [TiDB Community Architecture](https://github.com/pingcap/community/tree/master/architecture) we described the general requirements for role promotions. Please follow these principles.

Reply:
updated